File: Practice/새 폴더/f.ts
// const fs = require("fs");
// const filePath = process.platform === "linux" ? "/dev/stdin" : "../input.txt";

// let [count, ...input] = fs.readFileSync(filePath).toString().trim().split("\n");
// count = Number(count);
// input = input.map(Number);

// solution(input, count);

// function solution(input, count) {
//   const dp = Array(input.length).fill(0);
//   dp[0] = input[0];
//   dp[1] = input[0] + input[1];
//   dp[2] = Math.max(input[0], input[1]) + input[2];

//   for (let i = 3; i < input.length; i += 1) {
//     dp[i] = Math.max(dp[i - 2] + input[i], dp[i - 3] + input[i - 1] + input[i]);
//   }
//   console.log(dp[count - 1]);
// }

// const readline = require("readline");

// const rl = readline.createInterface({
//   input: process.stdin,
//   output: process.stdout,
// });

// rl.on("line", (line: number) => {
//   const MOD = 10007;

//   function a(n: number) {
//     let dp = new Array(n + 1).fill(0);
//     dp[1] = 1;
//     dp[2] = 2;

//     for (let i = 3; i <= n; i++) {
//       dp[i] = (dp[i - 1] + dp[i - 2]) % MOD;
//     }

//     return dp[n];
//   }
//   console.log(a(line));
//   rl.close();
// });

// rl.on("close", () => {
//   process.exit();
// });

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("", (N) => {
  N = parseInt(N);
  let count = 0;
  let num = 666;

  while (count < N) {
    if (String(num).includes("666")) {
      count++;
    }
    if (count < N) {
      num++;
    }
  }
  console.log(num);
  rl.close();
});

File: Practice/새 폴더/f.c

// function maxStairScore(numOfStairs, scores) {
//   if (numOfStairs === 1) {
//     return scores[0];
//   }

//   const dp = new Array(numOfStairs);
//   dp[0] = scores[0];
//   dp[1] = Math.max(scores[0] + scores[1], scores[1]);

//   for (let i = 2; i < numOfStairs; i++) {
//     dp[i] = Math.max(
//       dp[i - 2] + scores[i],
//       dp[i - 3] + scores[i - 1] + scores[i]
//     );
//   }

//   return dp[numOfStairs - 1];
// }
// const numOfStairs = parseInt(line);
// const scores = [];

// for (let i = 0; i < numOfStairs; i++) {
//   const score = parseInt(line);
//   scores.push(score);
// }

// const result = maxStairScore(numOfStairs, scores);
// console.log(result);